services/indexes/params/collections.go

```diff
@@ -285,12 +285,14 @@ func (p *ListTransactionsParams) Apply(b *dbr.SelectBuilder) *dbr.SelectBuilder
 
 type ListAssetsParams struct {
 	ListParams
-	ID    *ids.ID
-	Query string
-	Alias string
+	ID              *ids.ID
+	Query           string
+	Alias           string
+	EnableAggregate bool
+	PathParamID     string
 }
 
-func (p *ListAssetsParams) ForValue(q url.Values) error {
+func (p *ListAssetsParams) ForValues(q url.Values) error {
```

Comment:
Ugh. Good catch. Probably we should make `ListParams` not embedded so this can't happen.